src/utils/input_validation/validators.py

```diff
@@ -0,0 +1,177 @@
+                if guest_personal_id[1:3] == guest_birth_year[-2:]:
+                    if guest_personal_id[3:5] == guest_birth_mobth:
+                        if guest_personal_id[6:8] == guest_birth_day:
+                            return True
+        return False
+
+    if guest_country_code == "BE":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "BG":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "CZ":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "DK":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "DE":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "EE":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "IE":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "EL":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "ES":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "FR":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "BE":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "HR":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "IT":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "CY":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "LV":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "LT":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "LU":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "HU":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "MT":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "NL":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "AT":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "PL":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "PT":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "RO":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "SI":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "SK":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "FI":
+        print("Not implemented")
+        return False
+
+    if guest_country_code == "SE":
+        print("Not implemented")
+        return False
+
+def generate_random_username() -> str:
```

Comment:
This function is very different to all others in this file (i.e. generating a username vs. validation), I would suggest moving this function to another file.